content/post/democratize-data.md

```diff
@@ -0,0 +1,95 @@
+---
+title: "Strategies to Democratize Data"
+author: "Matt David"
+cover: ""
+date: 2020-03-19T20:35:46-04:00
+draft: true
+tags:
+ - Data Modeling
+ - Data Democratization
+ - Engineering
+ - Architecture
+---
+
+Talk to any executive and they will tell you that they want their organization to be more data driven.
+Ask how well they’re doing on this goal and you’ll find that most executives are disappointed with their progress so far.
+
+Data might be ubiquitous and quite cheap to store and compute, but it is largely inaccessible to most of their employees.
```

Comment:
This presumes that because data is inaccessible (this sentence) it provides less value to executives (the previous sentence). While it's a sufficient condition, I'm not sure it follows necessarily. (at least not obviously..?)